bindings/python/manifold3d.cpp

```diff
@@ -107,7 +107,7 @@ struct nb::detail::type_caster<glm::mat<C, R, T, Q>> {
   static handle from_cpp(glm_type mat, rv_policy policy,
                          cleanup_list *cleanup) noexcept {
     T *buffer = new T[R * C];
-    nb::capsule mem_mgr(buffer, [](void *p) noexcept { delete[] (T *)p; });
+    nb::capsule mem_mgr(buffer, [](void *p) noexcept { delete[](T *) p; });
```

Comment:
This is due to using the wrong version of clang-format. Make sure you've pulled master.

Reply:
I will try it again, thanks for pointing it out.

Reply:
Could you guide me on how to fix this, I had used the latest commit from master on the main repository.

Reply:
What clang-format version are you using? Use a newer one, e.g. 16.